packages/webapp/src/components/Task/AbandonTask/index.jsx

```diff
@@ -14,13 +14,21 @@ import Checkbox from '../../Form/Checkbox';
 import Rating from '../../Rating';
 import { getDateInputFormat } from '../../../util/moment';
 import { isNotInFuture } from '../../Form/Input/utils';
-
+import RadioGroup from '../../Form/RadioGroup';
+import styles from './styles.module.scss';
+import clsx from 'clsx';
+
+export const ABANDON_DATE_SELECTED = 'abandon_date_selected';
+export const ORIGINAL_DUE_DATE = 'original';
+export const TODAY_DUE_DATE = 'today';
+export const ANOTHER_DUE_DATE = 'another';
```

Comment:
@gursimran-singh 

If you could put these in a constants.js file that would fit with the structure of the app. AssignTask would be a good place to check.